src/pages/DatasetCatalog.jsx

```diff
@@ -0,0 +1,777 @@
+            </div>
+            {
+              (currentUser.isAdmin || currentUser.isChairPerson) && (
+                <button
+                  id="btn_addDataset"
+                  disabled={!currentUser.isAdmin && !currentUser.isChairPerson}
+                  onClick={() => props.history.push('data_submission_form')}
+                  className={`f-right btn-primary ${color}-background search-wrapper`}
+                  data-tip="Add a new Dataset"
+                  data-for="tip_addDataset"
+                >
+                  Add Dataset
+                  <span className="glyphicon glyphicon-plus-sign" style={{ marginLeft: '5px' }} aria-hidden="true" />
+                </button>
+              )
+            }
+          </div>
+        </div>
+        <div style={Theme.lightTable}>
+          <form className="pos-relative">
+            <div className={`${style['checkbox']} ${style['display-inline-block']}`} style={{ marginLeft: '28px', marginBottom: '0px' }}>
+              <input
+                checked={allOnPageSelected()}
+                type="checkbox"
+                //select-all="true"
+                className="checkbox-inline"
+                id="chk_selectAll"
+                onChange={selectAllOnPage}
+              />
+              <label className="regular-checkbox" htmlFor="chk_selectAll">
+                Select All Visible
+              </label>
+            </div>
+            <div className={style['display-inline-block']} style={{ top: '4px', position: 'absolute' }}>
+              <div style={{ borderLeft: '1px solid rgb(204, 204, 204)', height: '15px' }} />
+            </div>
+            <div className={`${style['checkbox']} ${style['display-inline-block']}`} style={{ marginLeft: '18px', marginBottom: '0px' }}>
+              <input
+                type="checkbox"
+                id="chk_onlySelected"
+                checked={filterToOnlySelected}
+                onChange={() => {
+                  setFilterToOnlySelected(!filterToOnlySelected);
+                }}
+              />
+              <label id="lbl_onlySelected" className={style['regular-checkbox']} htmlFor="chk_onlySelected">
+                {`Show ${numDatasetsSelected} Dataset${numDatasetsSelected !== 1 ? 's' : ''} Selected`}
+              </label>
+            </div>
+          </form>
+          <div className={currentUser.isAdmin ? style['table-scroll-admin'] : style['table-scroll']}>
+            <table className="table">
+              <thead>
+                <tr>
+                  <th />
+                  {currentUser.isAdmin || currentUser.isChairPerson && (
+                    <th className={style['cell-size']} style={{ minWidth: '14rem' }}>Actions</th>
+                  )}
+                  <th className={style['cell-size']}>{getSortDisplay({ field: 'datasetIdentifier', label: 'Dataset ID' })}</th>
+                  <th className={style['cell-size']}>{getSortDisplay({ field: 'Dataset Name' })}</th>
```

Comment:
There is an off-by-one error somewhere here for the admin case. Note in the screenshot how the Dataset ID column has actions and the Dataset Name column has the ID, etc.

## Admin Case
![Screenshot 2024-04-29 at 8 24 02 AM](https://github.com/DataBiosphere/duos-ui/assets/116679/ee45f056-492c-4a8e-b799-667b0c0625ae)

## Non-Admin case
![Screenshot 2024-04-29 at 8 27 27 AM](https://github.com/DataBiosphere/duos-ui/assets/116679/921f99d6-777a-4a6b-923e-42d4e5dc8e50)

Reply:
@rushtong  Thanks for catching this.